src/styles/main.scss

```diff
@@ -1,3 +1,23 @@
-body {
+@import "./stars.scss";
+@import "./card.scss";
+@import "./_variables.scss";
+@import "./nav.scss";
+@import "./catalog.scss";
+@import "./header.scss";
+
+* {
```

Comment:
Pls move all styles, defined in this file to [reset.scss](https://blog.hubspot.com/website/css-reset)